src/libs/actions/IOU.js

```diff
@@ -530,7 +531,7 @@ function createDistanceRequest(report, participant, comment, created, transactio
             createdChatReportActionID,
             createdIOUReportActionID,
             reportPreviewReportActionID: reportPreviewAction.reportActionID,
-            waypoints: JSON.stringify(transaction.comment.waypoints),
+            waypoints: JSON.stringify(validatedWaypoints),
```

Comment:
NAB: No need to introduce new variable for one usage. Use inline
```suggestion
            waypoints: JSON.stringify(TransactionUtils.validateWaypoints(transaction.comment.waypoints)),
```